lib/rust/ensogl/core/src/system/gpu/data/buffer/usage.rs

```diff
@@ -1,5 +1,8 @@
 //! `BufferUsage` specifies the intended usage pattern of the data store for optimization purposes.
 
+// === Non-Standard Linter Configuration ===
+#![allow(clippy::derivable_impls)]
```

Comment:
I got lost in macros internals and couldn't place this attribute in the macro-generated code.